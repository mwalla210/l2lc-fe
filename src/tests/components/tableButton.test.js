import React from 'react'
import renderer from 'react-test-renderer'
import TableButton from '../../components/tableButton'
jest.mock('../../components/buttonPrimary')

describe('TableButton', () => {
  it ('Renders', () => {
    let options = {
      onClick: jest.fn(),
      title: 'Button'
    }
    const component = renderer.create(
      <TableButton {...options}/>,
    )
    let tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
