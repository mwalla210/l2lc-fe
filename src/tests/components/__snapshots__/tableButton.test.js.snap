// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TableButton Renders 1`] = `null`;
